platform/discord/compose.yaml

```diff
@@ -2,7 +2,8 @@ services:
   discord:
     build:
       context: .
-      target: final
+      tags:
+        - "pocketsizefund/discord:latest"
```

Comment:
Rename the image (if possible) for consistency.

Reply:
yep, done